test/fixtures/gzip-joined

```diff
@@ -0,0 +1,19 @@
+0
```

Comment:
I'm not sure this file needed. You can extract content with node's zlib. It supports streamed format. If travis vails - just bump node version in config.

Reply:
ok, done